### PR TITLE
Change segment in AFQ.segmentation to use nib.tractogram

### PR DESCRIPTION
**Pull request by aarya22**

### CODE REVIEW

**Comment by aarya22** [comment]
@arokem take a look

**Comment by arokem** [comment]
Thanks. LGTM. Let's wait for Travis to run the tests, and then I can go ahead and merge.

File: AFQ/segmentation.py [hunk 1]
```diff
@@ -124,7 +123,7 @@ def segment(fdata, fbval, fbvec, streamlines, bundles,
         if as_generator:
             fiber_groups[bundle] = select_sl
         else:
-            fiber_groups[bundle] = list(select_sl)
+            fiber_groups[bundle] = nib.streamlines.Tractogram(select_sl, affine_to_rasmm=img.affine)
```
Comment (arokem) [inline AFQ/segmentation.py]: PEP8: please cut this line down to <80 chars

Comment (arokem) [reply AFQ/segmentation.py]: You can rewrite this as: 

```
fiber_groups[bundle] = nib.streamlines.Tractogram(
                            select_sl,
                            affine_to_rasmm=img.affine)
```

File: AFQ/utils/models.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import dipy.core.gradients as dpg
 
 
+#@profile
```
Comment (arokem) [inline AFQ/utils/models.py]: Please remove profiling decorator

File: AFQ/utils/models.py [hunk 3]
```diff
@@ -40,18 +41,17 @@ def prepare_data(data_files, bval_files, bvec_files, mask=None,
...
 
-    data = []
-    bvals = []
-    bvecs = []
-    for dfile, bval_file, bvec_file in zip(data_files, bval_files, bvec_files):
-        img = nib.load(dfile)
-        data.append(img.get_data())
-        bvals.append(np.loadtxt(bval_file))
-        bvecs.append(np.loadtxt(bvec_file))
+    #print(nib.load(data_files[0]).shape)
```
Comment (arokem) [inline AFQ/utils/models.py]: Please remove this print statement

File: AFQ/utils/models.py [hunk 4]
```diff
@@ -40,18 +41,17 @@ def prepare_data(data_files, bval_files, bvec_files, mask=None,
...
+    data = [nib.load(dfile).get_data() for dfile in data_files]
+    img = nib.load(data_files[-1])
+    bvals = [np.loadtxt(bval_file) for bval_file in bval_files]
+    bvecs = [np.loadtxt(bvec_file) for bvec_file in bvec_files]
+
+    gtab = dpg.gradient_table(bvals,bvecs,b0_threshold=b0_threshold)
 
     return img, data, gtab, mask
+
+# prepare_data(data_files='/Users/aarya/.dipy/stanford_hardi/HARDI150.nii.gz',
```
Comment (arokem) [inline AFQ/utils/models.py]: Please remove these lines

**Comment by aarya22** [comment]
Fixed the suggestions. Not sure why Travis is still failing.

**Comment by arokem** [comment]
You can see the test failures here:
https://travis-ci.org/yeatmanlab/pyAFQ/jobs/295861462

**Comment by aarya22** [comment]
I fixed the problems that was causing Travis to fail previously. Not sure what the problem is [now](https://travis-ci.org/yeatmanlab/pyAFQ/jobs/296363460). It says all passed on the bottom.

**Comment by arokem** [comment]
We automatically check for PEP8:
https://travis-ci.org/yeatmanlab/pyAFQ/jobs/296363460#L579

**Comment by aarya22** [comment]
Looks like test_segmentation.py returns a "CoverageException: Doesn't seem to be a coverage.py data file". Not sure what that means.

**Comment by arokem** [comment]
Not sure either. Did you try googling that error?

**Comment by richford** [comment]
Just trying to clean up old PRs and issues here. It looks like the build failed because of a flake8 error:

```
./AFQ/segmentation.py:127:80: E501 line too long (89 > 79 characters)
```

The coverage thing is a warning that I got on the builds for #63 but it wasn't fatal to the build.

**Comment by arokem** [comment]
This was superseded by #65

**Pull request closed without merging**